packages/core/src/alert/alert-group.stories.ts

```diff
@@ -43,8 +47,12 @@ export const alertGroup = () => {
   return html`
     <div cds-layout="vertical gap:sm">
       <cds-alert-group status="info" aria-label="This is an example info alert group">
-        <cds-alert closable>
-          This example is an alert a user may be able to close inside an alert group with a status of "info".
+        <cds-alert
+          closable
+          aria-label="This is an example info alert group"
+          cds-i18n='{ "closeButtonAriaLabel": "close my example alert"}'
```

Comment:
fwiw, an example that also showed how to change the default string of the alert icon would make accessibility very happy.